c10/core/impl/PyInterpreter.h

```diff
@@ -30,6 +30,46 @@ using Stack = std::vector<c10::IValue>;
 namespace c10 {
 namespace impl {
 
+struct C10_API PyInterpreter;
```

Comment:
Hmm, shouldn't it be renamed to something other than `PyInterpeter`, as name a bit misleading (i.e. this is just a opaque `TracerContext` pointer, isn't it?)

Reply:
It *is* a PyInterpreter though, we allocate one per python interpreter (there can be multiple in torchdeploy)